package.json

```diff
@@ -22,7 +22,7 @@
     "@types/react-dom": "18.2.4",
     "autoprefixer": "10.4.14",
     "framer-motion": "^10.16.1",
-    "next": "^14.0.4",
+    "next": "^14.2.5",
```

Comment:
maybe revert this one. 
we don't want the dependency bump along with the feature dev, unless necessary